### PR TITLE
case 19636 - Make path to VCPKG relative  to ENV variable

### PR DESCRIPTION
**Pull request by NissimHadar**
https://highfidelity.fogbugz.com/f/cases/19636/Auto-Tester-build-script-references-explicit-path

# Description
The path to OpenSSL has been changed from a fixed path to a path relative to VCPKG_ROOT.

# Testing
1.  Perform clean build and verify no errors.
2.  Verify that Auto-tester has been built.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/14494/HighFidelity-Beta-PR14301-959c544.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/14494/HighFidelity-Beta-PR14301-959c544.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/14494/HighFidelity-Beta-Interface-PR14301-959c544.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/14494/HighFidelity-Beta-PR14301-959c544.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/14494/HighFidelity-Beta-Interface-PR14301-959c544.exe

File: tools/auto-tester/CMakeLists.txt [hunk 1]
```diff
@@ -57,10 +57,10 @@ if (WIN32)
     )
         
     # add a custom command to copy the SSL DLLs
-    add_custom_command(
+     add_custom_command(
```
Comment (AndrewMeadows) [inline tools/auto-tester/CMakeLists.txt]: ^^^ This looks like a bad accidental indentation.

Comment (NissimHadar) [reply tools/auto-tester/CMakeLists.txt]: Corrected.

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/14527/HighFidelity-Beta-PR14301-93cff8d.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/14527/HighFidelity-Beta-PR14301-93cff8d.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/14527/HighFidelity-Beta-Interface-PR14301-93cff8d.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/14527/HighFidelity-Beta-PR14301-93cff8d.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/14527/HighFidelity-Beta-Interface-PR14301-93cff8d.exe

**Pull request merged**